components/About.vue

```diff
@@ -1,36 +1,36 @@
 <template>
   <v-card>
     <v-card-title class="headline text-center">
-      <v-row align="center" justify="space-around">About us</v-row>
+      <v-row align="center" justify="space-around">{{ head }}</v-row>
     </v-card-title>
     <v-card-text>
-      <p class="text-body-1">
-        In this age of increasing automation, programming is not limited to only
-        computer science and related fields. Every competent engineer needs to
-        know how to program. This makes Club of Programmers one of the most
-        important clubs in the institute.
-      </p>
-      <p class="text-body-1">
-        Club of Programmers (or better known as COPS) understands this
-        responsibility and plays a crucial part in maintaining the coding
-        culture in the institute. Introductory workshops are held for freshers
-        to introduce them to the various facets of programming. Regular
-        competitions are held for the general student body to test and sharpen
-        their programming skills. COPS also has an essential role in the
-        successful conduction of Technex, the institute’s techno-management
-        festival. ICM(International Coding Marathon) event organised by COPS in
-        Technex attracts the maximum participation in the fest.
-      </p>
-      <p class="text-body-1">
-        Although the club has been functioning quite well as compared to other
-        clubs, still, a lot of things needs to be done. Some great initiatives
-        were taken by the current secretaries, including the formation of
-        smaller groups focussed on specific areas (competitive programming
-        group, development group and machine learning group) and the fresher’s
-        mentorship program. Still there are lot of things that the club can
-        improve upon. We hope that we are able to take this momentum forward.
-      </p>
+      <p class="text-body-1">{{ para1 }}</p>
+      <p class="text-body-1">{{ para2 }}</p>
+      <p class="text-body-1">{{ para3 }}</p>
```

Comment:
use of slots with props can be a better option as it's not necessary that there will be 3 paras everywhere